### PR TITLE
Oauth text fix - google calendar

### PR DESCRIPTION
**Pull request by Stefan365**
//test: `@oauth-gcalendar or @oauth-salesforce`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by Stefan365** [comment]
retest this please

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Connections - OAuth | Testing Google calendar OAuth connector 
Connections - OAuth | Testing Salesforce OAuth connector 
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by tplevko** [comment]
@Stefan365 is this fix necessary? Seems like the fix for [ENTESB-11282](https://issues.jboss.org/browse/ENTESB-11282) was provided and verified by QE.

**Comment by Stefan365** [comment]
@tplevko ENTESB-11282 has been put to state "Productization backlog" as unresolved. I think @ENTESB annotation should skip this test

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/other/settings-oauth.feature [hunk 1]
```diff
@@ -49,9 +49,9 @@ Feature: Settings OAuth
     Then check that given "Google Calendar" oauth settings are not filled in
 
   @reproducer
-  @settings-oauth-validate
-  @gh-4021
-  @gh-5891
+    @settings-oauth-validate
+    @gh-4021
+    @gh-5891
```
Comment (tplevko) [inline ui-tests/src/test/resources/features/other/settings-oauth.feature]: why the spaces?

File: ui-tests/src/test/resources/features/connections/oauth.feature [hunk 2]
```diff
@@ -145,8 +150,10 @@ Feature: Connections - OAuth
     And wait until integration "OAuth-Gcalendar-test" gets into "Running" state
 
     When sleep for "6000" ms
+#    Integration does not work: ENTESB-11282:
```
Comment (tplevko) [inline ui-tests/src/test/resources/features/connections/oauth.feature]: seems like all the tests passed, is this remark really true?

File: ui-tests/src/test/resources/features/connections/oauth.feature [hunk 3]
```diff
@@ -25,6 +25,7 @@ Feature: Connections - OAuth
       | Google Calendar | Test-Google-Calendar-connection |
       | Twitter         | Test-Twitter-connection         |
 
+#    works OK on local computer, seems to be some unstability issue with jenkins job.
```
Comment (tplevko) [inline ui-tests/src/test/resources/features/connections/oauth.feature]: seems like all the tests passed, is this remark really true?

File: ui-tests/src/test/resources/features/other/settings-oauth.feature (hunk 1 above)
Comment (Stefan365) [reply ui-tests/src/test/resources/features/other/settings-oauth.feature]: this happened after auto format of code

Comment (Stefan365) [reply ui-tests/src/test/resources/features/other/settings-oauth.feature]: I'll remove it manually

File: ui-tests/src/test/resources/features/connections/oauth.feature (hunk 2 above)
Comment (Stefan365) [reply ui-tests/src/test/resources/features/connections/oauth.feature]: removed

File: ui-tests/src/test/resources/features/connections/oauth.feature (hunk 3 above)
Comment (Stefan365) [reply ui-tests/src/test/resources/features/connections/oauth.feature]: removed

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**